Source/WebCore/rendering/RenderLayerCompositor.h

```diff
@@ -209,6 +209,7 @@ class RenderLayerCompositor final : public GraphicsLayerClient, public CanMakeCh
     // Update event regions, which only needs to happen once per rendering update.
     void updateEventRegions();
     void updateEventRegionsRecursive(RenderLayer&);
+    EventTrackingRegions getTouchEventRegions();
```

Comment:
If this is just for testing, name it thusly.

Reply:
Fixed.